daemon/input.c

```diff
@@ -65,6 +63,11 @@ typedef struct {
 static char *key = NULL;
 static int key_index = 0;
 
+static int fd = 0;
+static event_io_t *event_io = NULL;
+static event_timer_t *timer = NULL;
+static bool pin_entry_active = false;
+
```

Comment:
use prefix input_ for static variables, this would be easier to understand that those are global vars
in this submodule.

Reply:
Done